index.html

```diff
@@ -72,17 +76,23 @@ <h1>Experience</h1>
 
             <li>Qwiklabs Projects in Google Cloud Platform with Coursera:</li>
               <ul>
-                <li><a href="https://www.coursera.org/account/accomplishments/professional-cert/2BHSL9BZC4FS" target="_blank">Cloud Architect</a></li>
-                <li><a href="https://www.coursera.org/account/accomplishments/professional-cert/4F84T5T43F7Q" target="_blank">Cloud Network Engineer</a></li>
-                <li><a href="https://www.coursera.org/account/accomplishments/professional-cert/9T9E9S6H6FFW" target="_blank">Cloud Security Engineer</a></li>
+                <li><a href="https://www.coursera.org/account/accomplishments/professional-cert/2BHSL9BZC4FS" target="_blank">Preparing for Google Cloud Certification: Cloud Architect</a></li>
+                <li><a href="https://www.coursera.org/account/accomplishments/professional-cert/4F84T5T43F7Q" target="_blank">Google Cloud Network Engineer</a></li>
+                <li><a href="https://www.coursera.org/account/accomplishments/professional-cert/9T9E9S6H6FFW" target="_blank">Google Cloud Security Engineer</a></li>
                 <li><a href="https://www.coursera.org/account/accomplishments/professional-cert/YY86MV2KM93W" target="_blank">Google IT Support</a></li>
                 <li><a href="https://www.coursera.org/account/accomplishments/professional-cert/7GXM6NRUF9UG" target="_blank">Google IT Automation with Python</a></li>
               </ul>
             <li>Proficient knowledge in Python, C, JavaScript, Html, and Google Cloud Platform.</li>
           </ul>
 
       </section>
-
+      <!--The container where I'll add dinamic ul and li elements-->
```

Comment:
Comments (or Pseudo code) are a great way to keep track of bugs or warnings during development. In production, it is encouraged to remove the code unless absolutely necessary. Your code should be easy to read and understand already.